cmd/user-agent/scripts/setup_assets.sh

```diff
@@ -8,7 +8,13 @@
 rm -rf public/agent-js-worker/
 mkdir -p public/agent-js-worker/assets
 npm install
-cp -Rp node_modules/@trustbloc/agent-js-worker/dist/assets/* public/agent-js-worker/assets
+
+if [[ $(grep "@trustbloc-cicd/agent-sdk" "package.json") ]] ; then
+   cp -Rp node_modules/@trustbloc-cicd/agent-sdk/dist/assets/* public/agent-js-worker/assets
+else
+   cp -Rp node_modules/@trustbloc/agent-sdk/dist/assets/* public/agent-js-worker/assets
```

Comment:
Do we need `else` here? Under what condition it happens (don't account manual editing of `package.json`)?

Reply:
Currently, we have two different npm builds for the same repo.
@trustbloc-cicd/agent-sdk (snapshot)
@trustbloc/agent-sdk (release)

If you want to use the latest package you need `@trustbloc-cicd/agent-sdk`. 
For stable/ release you need @trustbloc/agent-sdk. 
It was done to simplify development. Otherwise, you would fix it everywhere every time when you move from `snapshot` to `release` version.

Reply:
I'm not sure how it simplifies development.

Reply:
If you want to switch between versions it is enough to edit the package.json.
e.g `"@trustbloc-cicd/agent-sdk": "^0.1.5-snapshot-f8a5cb6"` or `"@trustbloc/agent-sdk": "^0.1.5"`

Reply:
Well, yes.

But updating package.json is something you'd have to do anyway.